src/Aspire.Hosting.Azure/AzureDeployingContext.cs

```diff
@@ -174,7 +175,7 @@ private async Task<bool> TryDeployContainerImages(DistributedApplicationModel mo
         }
 
         // Generate a deployment-scoped timestamp tag for all resources
-        var deploymentTag = $"aspire-deploy-{DateTime.UtcNow:yyyyMMddHHmmss}";
+        var deploymentTag = $"{environmentName}-{DateTime.UtcNow:yyyyMMddHHmmss}";
```

Comment:
This is using the name associated with the Azure environment resource to disambiguate further between any images.

Since the AzureEnvironmentResource has a name based on the AppHost SHA, this will mean local deployments from different AppHosts will use a different tag which I think is probably a good thing.